physionet-django/console/templates/console/submitted_projects.html

```diff
@@ -156,6 +157,7 @@ <h6 id='project_name'></h6>
                   <td><a href="{% url 'public_profile' project.submitting_author.user.username %}">{{ project.submitting_author }}</a></td>
                   <td>{{ project.submission_datetime|date }}</td>
                   <td>{{ project.revision_request_datetime|date }}</td>
+                  <td>{{ project.latest_reminder |date }}</td>
```

Comment:
Not sure whether it matters or not, but for consistency perhaps remove the space in `project.latest_reminder |date`?